KlingonAssistant/data/mem-03-D.xml

```diff
@@ -438,7 +438,7 @@ As a time element, {DaHjaj:n} precedes any adverbials.[1, sec. 6.7]</column>
       <column name="entry_name">Damu'</column>
       <column name="part_of_speech">n</column>
       <column name="definition">crease</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">Falte (im Papier)</column>
```

Comment:
Yes, both intentional folds made on paper, and unintentional/random ones made by wrinkling or balling up a piece of paper, are called "creases" in English.